src/main/kotlin/Main.kt

```diff
@@ -1,3 +1,147 @@
+import java.util.Scanner
+
 fun main(args: Array<String>) {
-    println("Hello World!")
+    val scan = Scanner(System.`in`)
+    var archives: MutableList<Archive> = mutableListOf()
```

Comment:
```suggestion
    val archives: MutableList<Archive> = mutableListOf()
```
Список и так мьютабельный, `var` излишен